include/kframework/foundry.md

```diff
@@ -478,6 +478,21 @@ This rule then takes from the function call data the account using `#asWord(#ran
       requires SELECTOR ==Int selector ( "store(address,bytes32,bytes32)" )
 ```
 
+#### `symbolicStorage` - Makes the storage of the given address completely symbolic.
+
+```
+function symbolicStorage(address) external;
+```
+
+`foundry.call.symbolicStorage` will match when the `symbolicStorage` cheat code function is called.
+This rule then takes the address using `#asWord(#range(ARGS, 0, 32))` and makes its storage completely symbolic.
+
+```{.k .bytes}
+    rule [foundry.call.symbolicStorage]:
+         <k> #call_foundry SELECTOR ARGS => #loadAccount #asWord(#range(ARGS, 0, 32)) ~> #setSymbolicStorage #asWord(#range(ARGS, 0, 32)) ... </k>
```

Comment:
Here, you can use directly `#asWord(ARGS)` instead. Since `ARGS` is defined as `#range(LM, ARGSTART +Int 4, ARGWIDTH -Int 4)` in the `foundry.call` rule, and since the cheat code function has a single argument, then it means that ARGS will contain only the address that you use.